index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
Right ! Good idea @konstantinzolotarev trailpack-datastore can have this interface to prevent futur problem, because maybe other ORM trailpack already have this problem too